app/index.html

```diff
@@ -382,14 +363,15 @@
   function page_change(page_id) {
 
     $(document).trigger('page-changed', page_id);
-    ar = ['dashboard', 'all_projects', 'about', 'submit']
+    ar = ['dashboard', 'all_projects', 'news', 'about', 'submit']
```

Comment:
Put `var` before variable declaration to make it scoped to function body. Otherwise variable will be global (as now). To help you avoiding such errors you can add `"use strict";` directive on top of script or function body.